src/gui/qgsnewhttpconnection.h

```diff
@@ -103,6 +103,16 @@ class GUI_EXPORT QgsNewHttpConnection : public QDialog, private Ui::QgsNewHttpCo
 
   protected:
 
+    //! Index of wfsVersionComboBox
+    enum WfsVersionIndex
+    {
+      WFS_VERSION_MAX = 0,
+      WFS_VERSION_1_0 = 1,
+      WFS_VERSION_1_1 = 2,
+      WFS_VERSION_2_0 = 3,
+      WFS_VERSION_API_FEATURES = 4,
```

Comment:
This is what I mean, because http://docs.opengeospatial.org/is/17-069r3/17-069r3.html is Version 1.0
```suggestion
      WFS_VERSION_API_FEATURES_1_0 = 4,
```
if the situation with extensions becomes messy, we might need a capability flag later.

Reply:
done